examples/web/activity_script_streaming_batch.html

```diff
@@ -0,0 +1,175 @@
+                    <label for="YAGNA_APPKEY">YAGNA_APPKEY: </label>
+                    <input id="YAGNA_APPKEY" type="text" value="411aa8e620954a318093687757053b8d"/>
+                </div>
+                <div>
+                    <label for="YAGNA_API_BASEPATH">YAGNA_API_BASEURL: </label>
+                    <input id="YAGNA_API_BASEPATH" type="text" value="http://127.0.0.1:7465/activity-api/v1" />
+                </div>
+            </div>
+            <h3>Actions</h3>
+            <div class="row vertical">
+                <div>
+                    <label for="AGREEMENT_ID">AGREEMENT_ID: </label>
+                    <div class="row padding-0">
+                        <input id="AGREEMENT_ID" type="text" />
+                        <button onclick="createActivity()">Create Activity</button>
+                        <button disabled onclick="destroyActivity()">Destroy Activity</button>
+                    </div>
+                </div>
+            </div>
+            <div class="row vertical">
+                <div>
+                    <label for="COMMAND">RUN SHELL COMMAND: </label>
+                    <div class="row padding-0">
+                        <input id="COMMAND" type="text" />
+                        <button onclick="addToScript()">Add to script</button>
+                    </div>
+                </div>
+            </div>
+            <div class="script console">
+                <h3>Script
+                    <span>
+                        <button onclick="getState()">Get state</button>
+                        <button onclick="clearScript()">Clear</button>
+                        <button id="execute" disabled onclick="executeScript()">Execute</button>
+                    </span>
+                </h3>
+                <ul id="script"></ul>
+
+            </div>
+        </div>
+        <div class="col-6 border-left">
+            <div class="logs console">
+                <h3>Logs</h3>
+                <ul id="logs"></ul>
+            </div>
+            <div class="results console">
+                <h3>Results</h3>
+                <ul id="results"></ul>
+            </div>
+        </div>
+    </div>
+
+    <script>
+        let activity;
+        let commands = [
+            new yajsapi.Deploy(),
+            new yajsapi.Start(),
+            new yajsapi.Terminate()
+        ];
+        refreshScript();
+        function log(msg) {
+            const logs_el = document.getElementById('logs');
+            const li = document.createElement('li');
+            li.appendChild(document.createTextNode(`[${new Date().toISOString()}] ${msg}`));
+            logs_el.appendChild(li);
+        }
+        function appendResults(result) {
+            const results_el = document.getElementById('results');
+            const li = document.createElement('pre');
+            li.appendChild(document.createTextNode(result));
+            results_el.appendChild(li);
+        }
+        function refreshScript() {
```

Comment:
shouldn't `refreshScript` take `commands` as a parameter? using the global seems like an antipattern to me...

Reply:
Indeed. I'll fix it at https://github.com/golemfactory/yajsapi/issues/326 because I'm going to use these examples for ui testing